porcupine/default_keybindings.tcl

```diff
@@ -59,13 +61,10 @@ for {set i 1} {$i <= 9} {incr i} {
 
 # tab_closing plugin
 event add "<<TabClosing:XButtonClickClose>>" <Button-1>
-if {[tk windowingsystem] == "aqua"} {
-    # right-click is Button-2, no wheel-click (afaik)
-    event add "<<TabClosing:ShowMenu>>" <Button-2>
-} else {
-    # right-click is Button-3, wheel-click is Button-2
+# no wheel click on mac (afaik)
+if {[tk windowingsystem] != "aqua"} {
+    # Wheel-click is Button-2
     event add "<<TabClosing:ShowMenu>>" <Button-3>
```

Comment:
I think you deleted the wrong line. The following suggestion changes it so that there is no `<<TabClosing:ShowMenu>>` but `<<TabClosing:HeaderClickClose>>` is unchanged:
```suggestion
    event add "<<TabClosing:HeaderClickClose>>" <Button-2>
```